docs/execution_flows.md

```diff
@@ -35,19 +35,19 @@ title: Safe{Core} Protocol High-level Plugin execution flow
 ---
 flowchart TD;
```

Comment:
This diagram reports error during rendering now. https://github.com/safe-global/safe-core-protocol/blob/feat/remove-isafe-from-function-parameters/docs/execution_flows.md#transaction-execution-through-plugin

Reply:
https://github.com/safe-global/safe-core-protocol/pull/100/commits/3545679bcc51718ab3e7ec71bf73fb55c38f74b4